operator/config/watcher/watcher.yaml

```
@@ -0,0 +1,23 @@
+# labels and selectors will be added using kustomize
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Watcher
+metadata:
+  name: watcher
+spec:
+  componentName: lifecycle-manager
+  contractVersion: v1
+  gvrsToWatch: [] # needs to be removed after removing it in the runtime-WatcherCRD
+  serviceInfo:
+    serviceName: lifecycle-manager-event-service
```

```suggestion
    serviceName: lifecycle-manager-event-service
```
this is prejudiced against the prefix `lifecycle-manager`. You will have to write a custom patch to inject the correct name here. (see CERT-MANAGER section in `default`)